File: .py
class Employee:
    def __init__(self, name, age, service_year, salary):
        self.name = name
        self.age = age
        self.service_year = service_year
        self.salary = salary

    def getName(self):
        return self.name

    def getAge(self):
        return self.age

    def getServiceYear(self):
        return self.service_year

    def getSalary(self):
        return self.salary


e = Employee("John", 30, 5, 50000)
print(e.getName(), e.getSalary())

File: .cpp
#include <iostream>
using namespace std;

class Employee {
private:
    string name;
    int age, serviceYear;
    double salary;

public:
    Employee(string n, int a, int s, double sal) {
        name = n; age = a; serviceYear = s; salary = sal;
    }

    ~Employee() {}

    string getName() { return name; }
    int getAge() { return age; }
    int getServiceYear() { return serviceYear; }
    double getSalary() { return salary; }
};

int main() {
    Employee e("John", 30, 5, 50000);
    cout << e.getName() << " " << e.getSalary();
    return 0;
}
